python/remote_api/src/lynx/kite.py

```diff
@@ -884,6 +884,39 @@ def f(**kwargs):
     _add_documentation_to_operation(f, name)
     return f
 
+  @property
+  def vertices(self) -> 'SingleOutputAtomicBox':
+    '''Returns the vertices as a table.'''
+    return self._select_all('vertices')
+
+  @property
+  def edges(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edges as a table.'''
+    return self._select_all('edges')
+
+  @property
+  def graph_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the graph attributes as a table.'''
+    return self._select_all('graph_attributes')
+
+  @property
+  def edge_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edge attributes as a table.'''
+    return self._select_all('edge_attributes')
+
+  def segmentations(self, name: str) -> 'SingleOutputAtomicBox':
+    '''Access a given segmentation by name.
```

Comment:
```suggestion
    '''Returns the named segmentation as a base project.
```

It's concise but specific enough to hint at the box. What do you think?

Reply:
I agree! Changed it.